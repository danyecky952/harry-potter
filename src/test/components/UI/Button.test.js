import React from 'react';
import { shallow } from 'enzyme';
import Button from '../../../components/UI/Button';
import '@testing-library/jest-dom';

describe('Prubeas en <Button />', () => {
  const wrapper = shallow(<Button />);
  test('debe mostrar el componente correctamente', () => {
    expect(wrapper).toMatchSnapshot();
  });

  test('debe de tener la clase ui-button', () => {
    const button = wrapper.find('button');

    expect(button.prop('className')).toBe('ui-button ');
  });
});
