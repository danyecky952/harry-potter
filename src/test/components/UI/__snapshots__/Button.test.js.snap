// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prubeas en <Button /> debe mostrar el componente correctamente 1`] = `
<button
  className="ui-button "
  onClick={[Function]}
/>
`;
